### PR TITLE
4.7 Docs corrections - config-settings.rst

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2150,7 +2150,7 @@ Includes three configuration keys: `ImageProxyType`, `ImageProxyURL` and `ImageP
 .. _documentation: https://docs.mattermost.com/administration/image-proxy
 
 +---------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"ImageProxyType": ""``, ``"ImageProxyURL": ""`` and ``"ImageProxyOptions": ""`` with string input.     |
+| This feature's ``config.json`` setting is ``"ImageProxyType": ""``, ``"ImageProxyURL": ""`` and ``"ImageProxyOptions": ""`` with string input.      |
```
Comment (amyblais) [inline source/administration/config-settings.rst]: I need help with formatting - this doesn't show up in preview/in the doc.

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2495,9 +2495,9 @@ Vary rate limit by user
 
 **False**: Rate limiting does not vary by user authentication token.
 
-+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature's ``config.json`` setting is ``"VaryByUser": false`` with options ``true`` and ``false`` for above settings respectively.                          |
-+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"VaryByUser": false`` with options ``true`` and ``false`` for above settings respectively.                              |
```
Comment (amyblais) [inline source/administration/config-settings.rst]: I need help with formatting - this doesn't show up in preview/in the doc.

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -3451,9 +3451,9 @@ Enable Preview Features (Experimental)
 
 **False**: Disables and hides preview features from **Account Settings** > **Advanced** > **Preview pre-release features**.
 
-+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| This feature’s ``config.json`` setting is ``"EnablePreviewFeatures": true`` with options ``true`` and ``false`` for above settings respectively.                     |
-+----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature’s ``config.json`` setting is ``"EnablePreviewFeatures": true`` with options ``true`` and ``false`` for above settings respectively.                    |
```
Comment (amyblais) [inline source/administration/config-settings.rst]: I need help with formatting - this doesn't show up in preview/in the doc.

**Comment by jasonblais** [comment]
@amyblais Fixed. For them to show up, the text needs to be wrapped inside a box as attached

image

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -2149,9 +2149,9 @@ Includes three configuration keys: `ImageProxyType`, `ImageProxyURL` and `ImageP
 
 .. _documentation: https://docs.mattermost.com/administration/image-proxy
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: @amyblais Is this formatted correctly?

Comment (amyblais) [reply source/administration/config-settings.rst]: I don't know how to format differently.

Comment (jasonblais) [reply source/administration/config-settings.rst]: @amyblais I'm not sure either, maybe Eric does?

Comment (amyblais) [reply source/administration/config-settings.rst]: @esethna Would you know how to format the link properly?

**Comment by lindy65** [comment]
Thanks @amyblais 

@esethna - can you check formatting please?

**Comment by amyblais** [comment]
I think this can be merged without further review. (?)

**Pull request merged**